server/index.js

```diff
@@ -45,6 +46,9 @@ tripRoute.initRoutes(apiRouter)
 const stageRoute = new StageRoute()
 stageRoute.initRoutes(apiRouter)
 
+const sharedRoute = new SharedRoute()
```

Comment:
I don't think we create an entire new router. Can just add the routes necessary to trip or stage routes and controller I think :')

Reply:
cant cause otherwise people trying to view share links would need to be logged in